src/main/sensors/gyro_init.c

```diff
@@ -72,6 +72,13 @@
 #include "sensors/gyro.h"
 #include "sensors/sensors.h"
 
+#if !defined(USE_GYRO_L3G4200D) && !defined(USE_GYRO_MPU3050) && !defined(USE_GYRO_MPU6050) && \
```

Comment:
We need to include `if defined(CLOUD_BUILD)` here

Reply:
Are you sure? The check is equally valid regardless.

Reply:
There is always a better way, but this fixes SITL build and make test.

Reply:
Adding `SIMULATOR_BUILD` is a better suggestion.

Reply:
Added `SIMULATOR_BUILD`.